content/doc/developer/publishing/requesting-hosting.adoc

```diff
@@ -62,12 +63,42 @@ Alternately, you can configure your plugin to be released automatically whenever
 In this case you only need write permission to GitHub; you never need local credentials.
 Follow link:/redirect/continuous-delivery-of-plugins[this guide] for details.
 
-== Categorize the Plugin
+=== Automate your GitHub repository
+
+==== Activate GitHub actions
+
+During the hosting process, you have included a shared workflow to generate changelogs automatically. Publishing changelogs as GitHub release notes is a good practice to keep your users informed about the changes in your plugin.
+
+To activate the workflow, you need to go to the `Actions` tab of your GitHub repository and click on the `I understand my workflows, go ahead and enable them` button.
```

Comment:
Here and elsewhere there are usages of backticks for things other than code. Backticks should only be used for inline code.